client/src/reducers/getMealListReducer.js

```diff
@@ -0,0 +1,16 @@
+import { GET_MEAL_LIST } from "../actions/actionTypes";
+
+const initialState = {};
+
+export default function(state = initialState, action) {
+  switch (action.type) {
+    case `${GET_MEAL_LIST}_FULFILLED`:
+      return { data: action.payload.data, isFulfilled: true };
```

Comment:
`isPending: False`, and let's have the pending type the first case cause it makes more sense it would be pending and then rejected or fulfiled